src/DataGrid.tsx

```diff
@@ -839,6 +839,10 @@ function DataGrid<R, SR, K extends Key>(
     }
 
     const column = columns[selectedPosition.idx];
+    if (column.renderEditCell == null || column.editable === false) {
```

Comment:
if `editable` is a function then we can check all the rows but I am not sure if it is worth